plotter/heat.go

```diff
@@ -178,16 +178,24 @@ func (h *HeatMap) DataRange() (xmin, xmax, ymin, ymax float64) {
 		xmax = 0.5
 		xmin = -0.5
 	default:
+		// Min and max are the first and last h.GridXYZ.X
+		// values extended beyond by half the distance
+		// between them and their adjacent h.GridXYZ.X
+		// value.
 		xmax = (3*h.GridXYZ.X(c-1) - h.GridXYZ.X(c-2)) / 2
```

Comment:
Make the code do more explicitly what is described in the comment? Then perhaps the comment is not even necessary.
```
xmax = h.GridXYZ.X(c-1) + (h.GridXYZ.X(c-1) - h.GridXYZ.X(c-2)) / 2
xmin = h.GridXYZ.X(0) - (h.GridXYZ.X(1) - h.GridXYZ.X(0)) / 2
```